corehq/blobs/migratingdb.py

```diff
@@ -45,3 +45,7 @@ def get_path(self, *args, **kw):
 
     def copy_blob(self, content, info, bucket):
         self.new_db.copy_blob(content, info, bucket)
+
+    def close(self):
```

Comment:
Yes, it's not part of the blob db interface. It's a convenience method on test blob db implementations only. I think it would be better to use [`TemporaryMigratingBlobDB`](https://github.com/dimagi/commcare-hq/blob/2553af0662c033080df4fd72083222836992a921/corehq/blobs/tests/util.py#L95), which has a `close()` method.